cht/adapter.py

```diff
@@ -40,48 +39,43 @@ def configure(self, config):
 			self.interfaces.append(interface)
 
 	def initialize(self, INIT_T):
-
-		self.preciceDt = self.precice.initialize()
-		if self.precice.is_action_required(precice.action_write_initial_data()):
+		self.precice.initialize()
+		if self.precice.requires_initial_data():
 			self.writeCouplingData(INIT_T)
-			self.precice.fulfilled_action(precice.action_write_initial_data())
```

Comment:
So, I modified the v2211 VM and I can start the tutorial, but it hits here:

```
---[precice] ERROR:  requiresInitialData() has to be called before initialize().
```

The `initialize()` needs to go after the `if`.